api/proto/teleport/clusterconfig/v1/access_graph.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.clusterconfig.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/clusterconfig/v1;clusterconfigv1";
+
+// AccessGraphConfigV1 holds the information about the configured Access Graph service.
+message AccessGraphConfigV1 {
+  // enabled indicates if the Access Graph is enabled for the cluster.
+  bool enabled = 1;
```

Comment:
Do we *always* upsert this resource and have it be true/false, or do we delete this resource once auth server is started with access_graph disabled? In the second case, the bool would be redundant, as we could simply treat `NotFoundError` from the backend as "disabled"

Reply:
I prefer to not use error codes to describe real behavior. It's much simpler to return the same structure with enabled than figuring out what does NotFoundError means in this context.

We do not upsert the resource, we read it from the service config.